src/Node/ReadLine.js

```diff
@@ -31,6 +31,18 @@ exports.prompt = function (readline) {
     };
 };
 
+exports.question = function(text) {
+    return function(callback) {
+        return function(readline) {
+            return function() {
+                readline.question(text, function(result) {
+                    callback(result)();
+                });
+            }                       
```

Comment:
Could you please add trailing semicolons to these `return` statements, to be consistent?